ci/jenkins/test-vmc.sh

```diff
@@ -353,6 +353,8 @@ function deliver_antrea {
     # Clean up dangling images generated in previous builds. Recent ones must be excluded
     # because they might be being used in other builds running simultaneously.
     docker image prune -f --filter "until=1h" > /dev/null
+    docker images | grep 'antrea' | awk '{print $3}' | xargs -r docker rmi || true
+    docker images | grep '<none>' | awk '{print $3}' | xargs -r docker rmi || true
```

Comment:
how do we know that these are unused, and if they are unused, why don't `docker image prune` take care of them?

Reply:
If cleaning up dangling images unused in the last hour doesn't free up sufficient disk space, we will have to remove all dangling and antrea images. Although removing intermediate images may result in slower image compilation, it helps prevent job failures due to insufficient storage.